src/status_im/contexts/profile/contact/header/view.cljs

```diff
@@ -55,4 +62,18 @@
           :icon-left :i/add-user}
          (i18n/label :t/send-contact-request)]]
 
+       (= contact-request-state constants/contact-request-state-received)
+       [rn/view {:style style/button-wrapper}
+        [quo/button
+         {:on-press  on-contact-review
+          :icon-left :i/add-user}
+         (i18n/label :t/contact-request-review)]]
+
+       (= contact-request-state constants/contact-request-state-mutual)
+       [rn/view {:style style/button-wrapper}
```

Comment:
`quo/button` supports the `container-style` prop. Have you tried to use that instead of wrapping in another view?

Reply:
Thank you for catching this, this works great 🙌